produce/lib/produce/merchant.rb

```diff
@@ -72,12 +58,50 @@ def app_identifier
       Produce.config[:app_identifier].to_s
     end
 
+    def pluralize(singular, arr)
+      return singular if arr.count == 1
+
+      "#{singular}s"
+    end
```

Comment:
This would be a great method to have in a monkey patch on Array in `fastlane_core` don't worry about making any changes to it now, just a thought that we use this type of logic all over and it would be nice to have a central place for it! 💯